packages/cli/init/src/initializeDocs.ts

```diff
@@ -24,10 +26,41 @@ export async function initializeDocs({
         taskContext
     });
 
-    await writeFile(
-        join(createDirectoryResponse.absolutePathToFernDirectory, RelativeFilePath.of(DOCS_CONFIGURATION_FILENAME)),
-        yaml.dump(getDocsConfig(createDirectoryResponse.organization))
-    );
+    if (createDirectoryResponse.absolutePathToFernDirectory) {
+        const docsYmlPath = join(
+            createDirectoryResponse.absolutePathToFernDirectory,
+            RelativeFilePath.of(DOCS_CONFIGURATION_FILENAME)
+        );
+
+        try {
```

Comment:
we have a doesFileExist in `@fern-api/fs-utils` which may clean up this code since you dont have to try catch